Source/WebCore/accessibility/ios/AccessibilityObjectIOS.mm

```diff
@@ -99,7 +99,11 @@
     if (!isSecureField())
         return 0;
 
-    auto* renderObject = downcast<AccessibilityRenderObject>(*this).renderer();
+    auto* accessibilityRenderObject = dynamicDowncast<AccessibilityRenderObject>(*this);
+    if (!accessibilityRenderObject)
+        return 0;
+
+    auto* renderObject = accessibilityRenderObject->renderer();
```

Comment:
`CheckedPtr`?

Reply:
We're only calling a trivial getter on renderObject (node()) so I don't think it's critical but we can do it, this is likely not perf critical.